CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
  
+## [1.3.1] - 2017-05-18
+### Changed
+- Make DRU order deterministic, from @wyegelwel
```

Comment:
The default cycle time for killing tasks that exceed max run time was dropped from 10 minutes to 1 minute as well